lib/board.rb

```diff
@@ -0,0 +1,73 @@
+
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+		reset!
```

Comment:
I really didn't get on board with reset! so what you suggested makes more sense to me now.